Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -476,18 +476,16 @@ void AODProducerWorkflowDPL::fillTrackTablesPerCollision(int collisionID,
   /// Add strangeness tracks to the table
   auto sTracks = data.getStrangeTracks();
   for (auto& collStrTrk : mCollisionStrTrk) {
-    if (collStrTrk.first < collisionID) {
+    if (mVtxToTableCollID.find(collStrTrk.first) == mVtxToTableCollID.end() || mVtxToTableCollID[collStrTrk.first] != collisionID) {
```

Comment:
@mpuccio @ddobrigk if I get it right, for every `collisionID` and every strange track the whole `mVtxToTableCollID` is scanned, which creates `Ncollisions^2 * Nstrange / 2` combinatorics. Isn't it better to fill in advance a
`unordered_map<int collID, std::vector strange_track IDs>` ?